charts/newrelic-infrastructure-beta/templates/control-plane/daemonset.yaml

```diff
@@ -0,0 +1,213 @@
+  {{- end }}
+spec:
+  {{- with .Values.updateStrategy }}
+  updateStrategy: {{ toYaml . | nindent 4 }}
+  {{- end }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "newrelic.name" . }}-control-plane
+      app.kubernetes.io/instance: {{ .Release.Name }}-control-plane
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        app.kubernetes.io/name: {{ include "newrelic.name" . }}-control-plane
+        app.kubernetes.io/instance: {{ .Release.Name }}-control-plane
+        {{- with .Values.podLabels }}
+          {{- toYaml . | nindent 8 }}
+        {{- end }}
+        mode: {{ include "newrelic.mode" . }}
+    spec:
+      {{- with .Values.images.pullSecrets }}
+      imagePullSecrets:
+          {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ template "newrelic.serviceAccountName" . }}-control-plane
+      {{- if .Values.privileged }}
+      hostNetwork: true
+      dnsPolicy: ClusterFirstWithHostNet
+      {{- end }}
+      containers:
+        - name: control-plane
+          image: {{ .Values.images.scraper.repository }}:{{ .Values.images.scraper.tag | default .Chart.AppVersion }}
+          imagePullPolicy: {{ .Values.images.scraper.pullPolicy }}
+          securityContext:
+            runAsUser: {{ .Values.runAsUser }}
+            runAsGroup: {{ .Values.runAsGroup }}
+            allowPrivilegeEscalation: false
+            readOnlyRootFilesystem: true
+          env:
+            - name: "NRIA_OVERRIDE_HOSTNAME_SHORT"
+              valueFrom:
+                fieldRef:
+                  apiVersion: "v1"
+                  fieldPath: "spec.nodeName"
+            - name: "NRI_KUBERNETES_HTTPSERVERPORT"
+              value: "8001"
+            - name: "NRI_KUBERNETES_CLUSTERNAME"
+              value: {{ include "newrelic.cluster" . | required "cluster name must be set" }}
+            - name: "NRI_KUBERNETES_NODENAME"
+              valueFrom:
+                fieldRef:
+                  apiVersion: "v1"
+                  fieldPath: "spec.nodeName"
+            - name: "NRI_KUBERNETES_NODEIP"
+              valueFrom:
+                fieldRef:
+                  apiVersion: "v1"
+                  fieldPath: "status.hostIP"
+            {{- if .Values.verboseLog }}
+            - name: NRI_VERBOSE
+              value: "1"
+            {{- end }}
+            {{- with .Values.controlPlane.extraEnv }}
+            {{- tpl . $ | nindent 12 }}
+            {{- end }}
+          {{- with .Values.controlPlane.extraEnvFrom }}
+          envFrom:
+            {{- tpl . $ | nindent 12 }}
+          {{- end }}
+          volumeMounts:
+            - name: nri-kubernetes-config
+              mountPath: /etc/newrelic-infra/integrations.d/nri-kubernetes-config.yml
+              subPath: nri-kubernetes-config.yml
+            - mountPath: /tmp
+              name: tmpfs-tmp
```

Comment:
Do we use `/tmp` in the _integration_? @paologallinaharbur @gsanchezgavier

Reply:
not that i know , even calculated metrics are using in memory storage . Just found this that we might have the chance to burn 😀 https://github.com/newrelic/nri-kubernetes/tree/develop/src/storage

Reply:
I guess that this was the one needed for the `/tmp`

Reply:
https://github.com/newrelic/nri-kubernetes/pull/297

Reply:
Seems like we're still using it, I'd address this on a separate PR.